ci/README.md

```diff
@@ -9,22 +9,22 @@ The ingress controller (Traefik or Istio), cluster issuer, and cert-manager shou
 ### Usage
 
 ```
-Usage: ./ketch-ci.sh [-t --ketch-tag] [-o --pool] [-ig --ingress] [--endpoint] [-a --app] [-i --image] [-e --env] [-ig --ingress] [--registry-secret] [--ketch-yaml] [--procfile] [--skip-resource-creation]
+Usage: ./ketch-ci.sh [-t --ketch-tag] [-o --framework] [-ig --ingress] [--endpoint] [-a --app] [-i --image] [-e --env] [-ig --ingress] [--registry-secret] [--ketch-yaml] [--procfile] [--skip-resource-creation]
```

Comment:
shipa uses -k and --framework to specify a framework, I think we can follow the same convention
```
-k, --framework 
```

Reply:
Modified to `-k` in readme, ketch-ci script, and short flags (deploy/parameters.go).